Pipfile

```diff
@@ -26,6 +26,8 @@ httmock = "==1.2.6"
 "flake8-print" = "*"
 "flake8-tuple" = "*"
 python-dateutil = "*"
+moto = "*"
+google-compute-engine = "*"
```

Comment:
https://github.com/GoogleCloudPlatform/compute-image-packages/issues/262

Travis was failing to install boto into a virtualenv, presumably because it's on google cloud and they're doing something funky to boto in their debian images.  Options were `--use-site-packages`, which I didn't fancy or to add `google-compute-engine` as a dependency.